docusaurus/docs/React/custom-code-examples/channel-list-preview.mdx

```diff
@@ -0,0 +1,313 @@
+---
+id: channel_list_preview
+sidebar_position: 1
+title: ChannelList - List, Preview
+---
+
+import CustomPreview from '../assets/CustomPreview.png';
+
+## How-to Guide for Customizing the ChannelList UI
+
+To customize the UI for the [`ChannelList`](../core-components/channel-list.mdx) you need to focus on the two components that dictate the UI, the list and preview components. These are set via the optional `List` and `Preview` props, and if nothing is provided they default to
+[`ChannelListMessenger`](https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelList/ChannelListMessenger.tsx)
+and [`ChannelPreviewMessenger`](https://github.com/GetStream/stream-chat-react/blob/master/src/components/ChannelPreview/ChannelPreviewMessenger.tsx). 
+
+Here is a guide to build these two UI components to use to override the defaults.
```

Comment:
i don't think you meant to say "to use to override"? can delete the to use I'd say